deploy.sh

```diff
@@ -6,7 +6,7 @@ pip install --user awscli
 export PATH=$PATH:$HOME/.local/bin
 # needs AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY envvars 
 eval $(aws ecr get-login --region us-east-1 --no-include-email)
-docker tag keboola/docker-sandbox-ssl-proxy:latest 147946154733.dkr.ecr.us-east-1.amazonaws.com/keboola/docker-sandbox-ssl-proxy:$TRAVIS_TAG
-docker tag keboola/docker-sandbox-ssl-proxy:latest 147946154733.dkr.ecr.us-east-1.amazonaws.com/keboola/docker-sandbox-ssl-proxy:latest
-docker push 147946154733.dkr.ecr.us-east-1.amazonaws.com/keboola/docker-sandbox-ssl-proxy:$TRAVIS_TAG
-docker push 147946154733.dkr.ecr.us-east-1.amazonaws.com/keboola/docker-sandbox-ssl-proxy:latest
+docker tag keboola/docker-sandbox-ssl-proxy:latest 061240556736.dkr.ecr.us-east-1.amazonaws.com/keboola/docker-sandbox-ssl-proxy:$TRAVIS_TAG
```

Comment:
i'd love if it could be done this way - https://github.com/keboola/component-generator/blob/master/templates-common/deploy.sh#L25
i.e. always deploy to testing, in case of tagged commit deploy tag+latest to prod

Reply:
ok, I changed it to do that.  But I just realised that travis will need 2 different AWS keys

Reply:
ah, didn't occur to me either :/